python/lsst/daf/butler/configs/registry.yaml

```diff
@@ -9,5 +9,5 @@ registry:
     opaque: lsst.daf.butler.registry.opaque.ByNameOpaqueTableStorageManager
     dimensions: lsst.daf.butler.registry.dimensions.static.StaticDimensionRecordStorageManager
     collections: lsst.daf.butler.registry.collections.synthIntKey.SynthIntKeyCollectionManager
-    datasets: lsst.daf.butler.registry.datasets.byDimensions.ByDimensionsDatasetRecordStorageManager
+    datasets: lsst.daf.butler.registry.datasets.byDimensions.ByDimensionsDatasetRecordStorageManagerUUID
```

Comment:
OK, I will undo this, my idea was that this is only relevant for new repos, old repos that have integers will still be usable with the old manager, but you are right, it's better to have this transition planned more explicitly.

Reply:
I'm not entirely sure why we wouldn't make this change. It only affects people doing `butler create`. At the very least I hope we've done one entire jenkins (with ci_hsc) done using the UUID manager otherwise it's not been tested anywhere.

Reply:
My concern was more about establishing good practices than this particular change.